src/machines/lesson-player-machine.ts

```diff
@@ -20,11 +20,11 @@ interface PlayerStateSchema {
 
 export type PlayerStateEvent =
   | {type: 'VIEW'}
-  | {type: 'VIEW'; lesson: any}
+  | {type: 'VIEW'; lesson: any; viewer: any}
```

Comment:
I'd like to move more of the transition logic between states into the machine and I noticed that the viewer was `undefined` when these transitions occurred because the viewer was still loading in the background